recipes/gsl-lite/all/conanfile.py

```diff
@@ -23,11 +24,12 @@ class GslLiteConan(ConanFile):
     #  2. GSL_THROW_ON_CONTRACT_VIOLATION: a gsl::fail_fast exception will be thrown
     #  3. GSL_UNENFORCED_ON_CONTRACT_VIOLATION: nothing happens
     #
-    options = {
-        "on_contract_violation": ["terminate", "throw", "unenforced"]
-    }
+    options = {"on_contract_violation": [None, "terminate", "throw", "unenforced"]}
     default_options = {
-        "on_contract_violation": "terminate",
+        "on_contract_violation": None,
```

Comment:
has this pattern not come up before, where we decide to leave the value undefined instead? I think it achieves the same effect in the package ID ?